crates/bevy_ecs/src/change_detection.rs

```diff
@@ -725,6 +757,7 @@ mod tests {
 
     use super::DetectChanges;
     use super::DetectChangesMut;
+    use super::MutUntyped;
```

Comment:
Not important, but can you combine these imports?

Also, the Future Possibilities section in the PR desc is out of date.